.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+| Q | A
+| --- | ---
+| Bug fix? | yes
```

Comment:
An explanation why I kept "yes" as default here: The most common answer is "no". Keeping "yes" as default forces the creator of the PR to think about whether this should really be a no.